go/cmd/dolt/commands/commit.go

```diff
@@ -170,22 +170,14 @@ func performCommit(ctx context.Context, commandStr string, args []string, cliCtx
 		return 0, true
 	}
 
-	if _, ok := queryist.(*engine.SqlEngine); ok {
-		// We have a local context, so we can call the classic (unmigrated) log command.
-		return LogCmd{}.Exec(ctx, "log", []string{"-n=1"}, temporacyDEnv, nil), false
-	} else {
-		// TODO: when dolt log is migrated, remove this block printing out the commit and print with a dolt log call in Exec()
-		schema, rowIter, err = queryist.Query(sqlCtx, "select * from dolt_log() limit 1")
-		if err != nil {
-			cli.Println(err.Error())
-			return 1, false
-		}
-		err = engine.PrettyPrintResults(sqlCtx, engine.FormatTabular, schema, rowIter)
-		if err != nil {
-			cli.Println(err.Error())
-			return 1, false
-		}
-	}
+	// TODO: when dolt log is migrated, remove this block printing out the commit and print with a dolt log call in Exec()
```

Comment:
Drop this comment. Don't see why we would ever use log in the future if this works.